ex-1.sc

```diff
@@ -0,0 +1,20 @@
+def examWords(args: List[String]): Unit = {
+  val filteredArgs = args.filter(_.length > 2)
+  if (filteredArgs.nonEmpty) {
+    val maxWord = filteredArgs.maxBy(_.length)
+    val minWord = filteredArgs.minBy(_.length)
+
+    print("Longer word/s: ")
+    print(filteredArgs.filter(_.length.equals(maxWord.length)).mkString(", "))
+    print(s" with ${maxWord.length} letters")
+
+    println
+    
+    print("Shorter word/s: ")
```

Comment:
I rathen calculate the all results and have a single `println(s"""Shorter word/s: $ShortestByComma with ${minWord.length} letters""")`